NodeOS.svg

```diff
@@ -3,6 +3,7 @@
    xmlns="http://www.w3.org/2000/svg"
    width="56cm"
    height="60cm"
+   shape-rendering="geometricPrecision"
```

Comment:
It should force to output borders with high contrast, reducing the numbers
of colours. If that's not enlught, then we'll need to reduce the palette of
the PNG image.

El 9/8/2016 16:30, "John Green" <notifications@github.com> escribió:

> In NodeOS.svg
> <https://github.com/NodeOS/media/pull/3#discussion_r74069206>:
>
> > @@ -3,6 +3,7 @@
> >     xmlns="http://www.w3.org/2000/svg"
> >     width="56cm"
> >     height="60cm"
> > +   shape-rendering="geometricPrecision"
>
> It outputs a really nice image compared to what I've got before. I'll test
> it with crispEdges again, but I'm not use if crispEdges really improves the
> quality of the picture
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/NodeOS/media/pull/3/files/231d44d1dcfea0e3485b9048a4c589cd61cfc889#r74069206>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAgfvqVaB_n6MG84BSfgYiJ8wCidwXdZks5qeI70gaJpZM4JgHOI>
> .
>